src/portal/src/app/project/list-project/list-project.component.spec.ts

```diff
@@ -14,14 +14,15 @@ import { StatisticHandler } from "../../shared/statictics/statistic-handler.serv
 import { ConfirmationDialogService } from "../../shared/confirmation-dialog/confirmation-dialog.service";
 import { of } from 'rxjs';
 import { BrowserAnimationsModule, NoopAnimationsModule } from "@angular/platform-browser/animations";
+import { delay } from 'rxjs/operators';
 describe('ListProjectComponent', () => {
     let component: ListProjectComponent;
     let fixture: ComponentFixture<ListProjectComponent>;
     const mockProjectService = {
         listProjects: () => {
             return of({
                 body: []
-            });
+            }).pipe(delay(0));
```

Comment:
you can open auto detect:  
        { provide: ComponentFixtureAutoDetect, useValue: true }